public/app/features/explore/ExploreToolbar.tsx

```diff
@@ -146,7 +146,11 @@ export class UnConnectedExploreToolbar extends PureComponent<Props> {
               ) : null}
 
               <Tooltip content={'Copy shortened link to the executed query'} placement="bottom">
-                <ToolbarButton icon="share-alt" onClick={() => createAndCopyShortLink(window.location.href)} />
+                <ToolbarButton
```

Comment:
Couldn't we solve this on Tooltip level? Because if we have Tooltip, it usually contains explaining content. Couldn't we just pass it to component it is wrapping? This way it will be synced and we won't have to add duplicated aria labels everywhere. Something like this in https://github.com/grafana/grafana/blob/main/packages/grafana-ui/src/components/Tooltip/Tooltip.tsx#L45:

```
            {React.cloneElement(children, {
              ref: tooltipTriggerRef,
              onMouseEnter: showPopper,
              onMouseLeave: hidePopper,
              'aria-label': controllerProps.content,
              title: controllerProps.content,
            })}
```

Reply:
While in this case could be a good idea, I don't think tooltips content is intended to replace labels and/or titles. 
Usually, in tooltips you find more detailed information or explainers regarding what an option does rather than just a "label".

what do you think?

Reply:
Yeah, you are right.